### PR TITLE
Dread: Ferenia revision (EMMI and Escue zone missing)

### PR DESCRIPTION
**Pull request by Hugoshido**
- Made all Walljumps (Beginner) and some movement Trivial
- Updated some tricks to templates
- Added Speedbooster movement
- Added "Disable Door Rando" to cold room 
-

### CODE REVIEW

**Review by XenoWars — CHANGES_REQUESTED** [review]

**Comment by XenoWars** [comment]
Quite a bit, though I would have preferred waiting until I did the IBJ changes as I've been saying in Discord.

File: randovania/games/dread/json_data/Ferenia.txt [hunk 1]
```diff
@@ -1189,7 +1186,7 @@ Extra - asset_id: collision_camera_015
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
   > Tile Group (POWERBEAM) 2
-      Simple IBJ or Use Spin Boost
+      Trivial
```
[the remark this answers is not in the document]
Comment (Hugoshido) [reply randovania/games/dread/json_data/Ferenia.txt:1189]: Yes you can just Walljump up there

File: randovania/games/dread/json_data/Ferenia.txt [hunk 2]
```diff
@@ -1490,14 +1484,10 @@ Extra - asset_id: collision_camera_018
...
       All of the following:
-          Speed Booster
+          Speed Booster and Disabled Door Lock Randomizer
           Any of the following:
               Gravity Suit
               Heat/Cold Runs (Intermediate) and Cold Damage ≥ 50
-          Any of the following:
-              # Temporary until door rando
-              Charge Beam
-              Knowledge (Beginner) and Lay Power Bomb
```
[the remark this answers is not in the document]
Comment (Hugoshido) [reply randovania/games/dread/json_data/Ferenia.txt:1500]: Check

File: randovania/games/dread/json_data/Ferenia.txt [hunk 3]
```diff
@@ -1520,7 +1510,7 @@ Extra - asset_id: collision_camera_018
               Heat/Cold Runs (Intermediate) and Cold Damage ≥ 50
           All of the following:
               # Temporary until door rando
-              Grapple Beam
+              Grapple Beam and Disabled Door Lock Randomizer
```
[the remark this answers is not in the document]
Comment (Hugoshido) [reply randovania/games/dread/json_data/Ferenia.txt:1513]: Check

File: randovania/games/dread/json_data/Ferenia.txt [hunk 4]
```diff
@@ -3087,8 +3077,10 @@ Extra - asset_id: collision_camera_040
       Morph Ball and After Ferenia - Central Unit
   > Start Point
       Trivial
+  > Tile Group (POWERBOMB)
+      Flash Shift and Speed Booster
```
[the remark this answers is not in the document]
Comment (Hugoshido) [reply randovania/games/dread/json_data/Ferenia.txt:3081]: The idea was the speedmethod going from there.. But I added it to the central platform instead

File: randovania/games/dread/json_data/Ferenia.txt [hunk 5]
```diff
@@ -3087,8 +3077,10 @@ Extra - asset_id: collision_camera_040
...
   > Central Platform
-      Flash Shift
+      Flash Shift or Speed Booster or Simple IBJ
```
[the remark this answers is not in the document]
Comment (Hugoshido) [reply randovania/games/dread/json_data/Ferenia.txt:3083]: Check

File: randovania/games/dread/json_data/Ferenia.txt [hunk 6]
```diff
@@ -3158,13 +3150,13 @@ Extra - asset_id: collision_camera_040
   > Dock to Purple EMMI Introduction Access
       Trivial
   > Tile Group (POWERBOMB)
-      Space Jump or Infinite Bomb Jump (Beginner)
+      Space Jump or Simple IBJ
```
[the remark this answers is not in the document]
Comment (Hugoshido) [reply randovania/games/dread/json_data/Ferenia.txt:3153]: Check

File: randovania/games/dread/json_data/Ferenia.txt [hunk 7]
```diff
@@ -3158,13 +3150,13 @@ Extra - asset_id: collision_camera_040
...
   > Tile Group (WEIGHT) 1
       Can Slide
   > Tunnel to EMMI Zone Exit West
-      Any of the following:
-          Simple IBJ or Use Spin Boost
-          Flash Shift and Walljump (Beginner)
+      All of the following:
+          After Ferenia - Central Unit
+          Flash Shift or Spin Boost or Simple IBJ
```
[the remark this answers is not in the document]
Comment (Hugoshido) [reply randovania/games/dread/json_data/Ferenia.txt:3159]: Check

File: randovania/games/dread/json_data/Ferenia.txt [hunk 8]
```diff
@@ -390,7 +385,7 @@ Extra - asset_id: collision_camera_007
   * Extra - start_point_actor_name: teleporter_000_platform
   * Extra - start_point_actor_def: actordef:actors/props/weightactivatedplatform_teleport/charclasses/weightactivatedplatform_teleport.bmsad
   > Tunnel to Separate Tunnels Room (Lower)
-      Trivial
+      Morph Ball
```
[the remark this answers is not in the document]
Comment (Hugoshido) [reply randovania/games/dread/json_data/Ferenia.txt:388]: Check

File: randovania/games/dread/json_data/Ferenia.txt [hunk 9]
```diff
@@ -323,20 +321,17 @@ Extra - asset_id: collision_camera_006
...
       Trivial
 
 > Tunnel to Fan Room Access; Heals? False
   * Layers: default
   * Morph Ball Tunnel to Fan Room Access/Tunnel to Fan Room
   > Door to Cold Room (Small)
-      Speed Booster
+      Any of the following:
+          Speed Booster
+          Movement (Beginner) and Simple IBJ
```
[the remark this answers is not in the document]
Comment (Hugoshido) [reply randovania/games/dread/json_data/Ferenia.txt:334]: Check

File: randovania/games/dread/json_data/Ferenia.txt [hunk 10]
```diff
@@ -218,9 +217,7 @@ Extra - asset_id: collision_camera_004
   > Dock to Purple EMMI Introduction
       Trivial
   > Tunnel to Fan Room
-      Any of the following:
-          Grapple Beam or Space Jump or Speed Booster or Infinite Bomb Jump (Beginner)
-          Spider Magnet and Use Spin Boost
+      Flash Shift or Grapple Beam or Spider Magnet or Speed Booster or Simple IBJ or Use Spin Boost
```
[the remark this answers is not in the document]
Comment (Hugoshido) [reply randovania/games/dread/json_data/Ferenia.txt:220]: Yes you can Walljump + Flashshift up there..

File: randovania/games/dread/json_data/Ferenia.txt [hunk 11]
```diff
@@ -73,8 +73,8 @@ Extra - asset_id: collision_camera_002
       Morph Ball
   > Right Alcove
       All of the following:
-          Morph Ball
-          Grapple Beam or Simple IBJ or Use Spin Boost
+          Morph Ball and Speed Booster and Disabled Door Lock Randomizer
```
[the remark this answers is not in the document]
Comment (Hugoshido) [reply randovania/games/dread/json_data/Ferenia.txt:76]: Check

File: randovania/games/dread/json_data/Ferenia.txt [hunk 12]
```diff
@@ -126,11 +126,10 @@ Extra - asset_id: collision_camera_002
           Grapple Beam or Spider Magnet or Simple IBJ or Use Spin Boost
   > Ceiling Tunnel
       Any of the following:
-          Space Jump or Simple IBJ
+          Simple IBJ or Use Spin Boost
           All of the following:
-              Flash Shift
+              Flash Shift and Morph Ball
               Grapple Beam or Spider Magnet
```
[the remark this answers is not in the document]
Comment (Hugoshido) [reply randovania/games/dread/json_data/Ferenia.txt:132]: You can just jump + flash to the ledge and then up to the ceiling

File: randovania/games/dread/json_data/Ferenia.txt [hunk 13]
```diff
@@ -52,7 +52,7 @@ Extra - asset_id: collision_camera_001
   * Extra - right_shield_def: None
   > Dock to Purple EMMI Introduction
       Any of the following:
-          Grapple Beam or Space Jump
+          Grapple Beam or Space Jump or Simple IBJ
```
[the remark this answers is not in the document]
Comment (Hugoshido) [reply randovania/games/dread/json_data/Ferenia.txt:55]: Spin boost is enough to get up there

File: randovania/games/dread/json_data/Ferenia.txt [hunk 14]
```diff
@@ -988,7 +985,7 @@ Extra - asset_id: collision_camera_012
   > Underwater Ledge Left
       All of the following:
           Gravity Suit
-          Simple IBJ or Use Spin Boost
+          Flash Shift or Simple IBJ or Use Spin Boost
```
[the remark this answers is not in the document]
Comment (Hugoshido) [reply randovania/games/dread/json_data/Ferenia.txt:988]: Check

**Pull request closed without merging**